src/models/application.js

```diff
@@ -159,3 +171,31 @@ Application.redeploy = function(api, appId, orgaId) {
 
   return api.owner(orgaId).applications._.instances.post().withParams(params).send();
 };
+
+Application.setScalability = function(api, appId, orgaId, scalabilityParameters) {
+  Logger.info("Scaling the app: " + appId);
+
+  var s_app = Application.get(api, appId, orgaId).toProperty();
+  var s_body = s_app.map(function(app) {
+    var instance = _.clone(app.instance);
+
+    instance.minFlavor = instance.minFlavor.name;
+    instance.maxFlavor = instance.maxFlavor.name;
+
+    if (scalabilityParameters.minFlavor)
```

Comment:
if the new minimum (flavor|number) is greater than the current max, then lift the maximum value to match the new minimum value.